### PR TITLE
completed home header feature

### PR DESCRIPTION
**Pull request by Tanvir04khan**

### CODE REVIEW

**Review by faran4engg — COMMENTED** [review]
Add screenshot of the part you completed @Tanvir04khan

File: src/components/Home/header/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import "./_header.scss";
+import avatar from "../../../assets/images/pngwing.com.png";
+
+import { FaBars } from "react-icons/fa";
+import { AiOutlineSearch } from "react-icons/ai";
+import { MdNotifications, MdApps } from "react-icons/md";
+
+const header = () => {
```
Comment (faran4engg) [inline src/components/Home/header/Header.jsx]: component should be in upper case±!!!!!!!!

File: src/index.jsx [hunk 2]
```diff
@@ -1,8 +1,10 @@
 import React from "react";
 import ReactDOM from "react-dom";
-
 import App from "./App";
 
+import "bootstrap/dist/css/bootstrap.min.css";
```
Comment (faran4engg) [inline src/index.jsx:5]: all that border etc classes are coming from here !

**Review by faran4engg — APPROVED** [review]
Change that header -> `Header` and then merge

File: src/index.jsx (hunk 2 above)
Comment (Tanvir04khan) [reply src/index.jsx:5]: ok

File: src/components/Home/header/Header.jsx (hunk 1 above)
Comment (Tanvir04khan) [reply src/components/Home/header/Header.jsx]: changed component name

**Pull request merged**